src/engine/strat_engine/engine.rs

```diff
@@ -244,7 +246,9 @@ impl Engine for StratEngine {
             Err(err)
         } else {
             devlinks::pool_removed(&pool_name);
-            Ok(DeleteAction::Deleted(uuid))
+            let result = DeleteAction::Deleted(uuid);
+            info!("pool destroyed: uuid {}, name {}", result, pool_name);
+            Ok(result)
```

Comment:
At this point, this should be changed back to what was here before. We are now doing the logging at the D-Bus layer.

Reply:
Resolved.